packages/theme-a11y/README.md

```diff
@@ -20,3 +20,17 @@ If you prefer not to use a package manager, you can download the latest version
 - [theme-a11y.min.js](http://unpkg.com/@shopify/theme-a11y@latest/dist/theme-a11y.min.js)
 
 These files make Theme A11y accessible via the `Shopify.theme.a11y` global variable.
+
+---
+
+### accessibleLinks(messages, targetLinks, prefix)
+
+Add a preventive message to external links and links that open to a new window.
+This is being used to provide better accessibility experience.
```

Comment:
```suggestion

```

Sentence not needed - all of these helpers are here to provide a better experience 😄